### ISSUE TITLE
Drag scroll only works in the first slide

### ISSUE THREAD

**Issue by fahdaddi** [oruga-ui/oruga#338]
### Overview of the problem

**Oruga** version: [0.5.5]
**Nuxtjs** version: [2.15.8]
**OS/Browser**: Google Chrome Iphone Xs Max

### Description

the drag only works in the first item (check `www.optigura.fr` for example)

**Comment by jtommy** [comment]
What do you mean? I am able to drag all sliders in the homepage (android + Chrome)

**Comment by astagi** [comment]
@fahdaddi yep, I can reproduce it on iPhoneXR + Chrome (same problem using Safari). Carousel from docs works though.. need to investigate further..

**Comment by fahdaddi** [comment]
@astagi is there a way to debug the component in Xcode emulator? (something like the react-native debugger)

**Comment by astagi** [comment]
I don't know @fahdaddi :/ I use real devices, it could be real interesting to figure out how to debug components on mobile devices

**Comment by tonybebber** [comment]
> @fahdaddi yep, I can reproduce it on iPhoneXR + Chrome (same problem using Safari). Carousel from docs works though.. need to investigate further..

Were you ever able to figure out a solution? Dealing with the same problem on physical iPhone 12 Pro using Safari (using v0.5.5 of Oruga), where dragging works on the first carousel slide, but then dragging becomes completely disabled.
